telemetry/csharp/AwsToolkit.Telemetry.Events.Tests/Generated/GeneratedCode.cs

```diff
@@ -8240,6 +8289,9 @@ public struct Runtime
         /// dotnet5.0
         public static readonly Runtime Dotnet50 = new Runtime("dotnet5.0");
         
+        /// dotnet6.0
+        public static readonly Runtime Dotnet60 = new Runtime("dotnet6.0");
```

Comment:
The Lambda runtime enum value is `dotnet6`. Not sure if this code is meant match exactly that.

Reply:
In my understanding we do seem to that. Thanks for the callout!